examples/provider-install-verification/main.tf

```diff
@@ -58,7 +58,7 @@ resource "deno_domain_verification" "example_domain_verification" {
   }
 }
 
-resource "deno_certificate_provisioning" "example_certificate_provisioning" {
```

Comment:
I see no change in the resource name in the resource definition. Is this change intended to be included in this PR?